FeedStoreChallenge/Infrustructure/ManagedFeedImage.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  ManagedFeedImage.swift
+//  FeedStoreChallenge
+//
+//  Created by Zheng Kanyan on 2021/6/11.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedFeedImage)
+internal class ManagedFeedImage: NSManagedObject {
```

Comment:
`ManagedFeedImage` is not supposed to be inherited from, so it should be marked as final.
Also, the convention in this codebase is to omit the default access level of `internal`, so it can be removed.
```suggestion
final class ManagedFeedImage: NSManagedObject {
```